### PR TITLE
Update the past year's events #90

### PR DESCRIPTION
**Pull request by MadhushaPrasad**

### CODE REVIEW

**Review by stargazer39 — APPROVED** [review]
Looks fine. 😍

**Review by susithrupasinghe — APPROVED** [review]

**Comment by senuravihanjayadeva** [comment]
@MadhushaPrasad @stargazer39  Please changed the branch to development first

File: src/interfaces/EventInterface.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ interface IEvent {
     _id?: string;
     title: string;
     description: string;
-    dateTime: Date;
+    dateTime: any;
```
Comment (senuravihanjayadeva) [inline src/interfaces/EventInterface.ts:5]: Why this changed Date to any

File: src/pages/event/past-events.tsx [hunk 2]
```diff
@@ -1,118 +1,119 @@
...
-    if (keyboardCode === 37) {
-      slidePrev();
-    }
-  };
+    const [pastEventList, setPastEventList] = useState<IEvent[]>();
 
-  const [pastEventList, setPastEventList] = useState<IEvent[]>();
+    //fetch all past events
+    useEffect(() => {
+        // getPastEvents()
```
Comment (senuravihanjayadeva) [inline src/pages/event/past-events.tsx:41]: Remove unused code segments

File: src/data/PastEventData.json [hunk 3]
```diff
@@ -1,46 +1,67 @@
 {
-  "data": [
+  "EventData": [
```
Comment (senuravihanjayadeva) [inline src/data/PastEventData.json:2]: Google JSON Style Guide (recommendations for building JSON APIs at Google), It recommends that: Property names must be camelCased

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Pull request merged**